diff --git a/source/CMakeLists.txt b/source/CMakeLists.txt
index c3b5fde0..51c61c35 100644
--- a/source/CMakeLists.txt
+++ b/source/CMakeLists.txt
@@ -103,7 +103,7 @@ if(USE_CUDA_TOOLKIT)
     set(CMAKE_CUDA_COMPILER ${CUDAToolkit_NVCC_EXECUTABLE})
   endif()
   if(NOT DEFINED CMAKE_CUDA_HOST_COMPILER)
-    set(CMAKE_CUDA_HOST_COMPILER ${CMAKE_CXX_COMPILER})
+   #set(CMAKE_CUDA_HOST_COMPILER ${CMAKE_CXX_COMPILER})
   endif()
   add_definitions("-DGOOGLE_CUDA")
   message(STATUS "Found CUDA in ${CUDAToolkit_BIN_DIR}, build nv GPU support")